tools/tgstation-server/config.bat

```diff
@@ -0,0 +1,28 @@
+rem Server Tools configuration file. Lines starting with rem are comments and ignored.
+
+rem This must be set to the name of your dme without the .dme part. (should be fine leaving this alone unless you renamed the code)
+set PROJECTNAME=tgstation
+
+rem location of the repo.
+set REPO_URL=https://github.com/tgstation/-tg-station.git
+
+rem what map file to use. This should be the name of the dm, not dmm (and without the .dm part)
+set MAPFILE=metastation
```

Comment:
Oh, forgot to change it back from when I was testing that the build script
was overwriting the ticked map file correctly
On May 8, 2015 2:51 PM, "Iamgoofball" <notifications@github.com> wrote:

> In tools/tgstation-server/config.bat
> <https://github.com/tgstation/-tg-station/pull/9411#discussion_r29979876>:
>
> > @@ -0,0 +1,28 @@
> > +rem Server Tools configuration file. Lines starting with rem are comments and ignored.
> > +
> > +rem This must be set to the name of your dme without the .dme part. (should be fine leaving this alone unless you renamed the code)
> > +set PROJECTNAME=tgstation
> > +
> > +rem location of the repo.
> > +set REPO_URL=https://github.com/tgstation/-tg-station.git
> > +
> > +rem what map file to use. This should be the name of the dm, not dmm (and without the .dm part)
> > +set MAPFILE=metastation
>
> this needs to be box by default
>
> box is our default "guarenteed to work" map
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/9411/files#r29979876>.
>